### PR TITLE
cleaning up bq test functions

### PR DESCRIPTION
**Pull request by mmahsereci**
*Issue #172 , if available:* 

*Description of changes:*
For bq functions only. Will do the others in another PR. Return structure is: wrapped_func, integral_bounds
Also, I added one more bq test function.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by mmahsereci** [comment]
Good point. Nevertheless, integral_bounds is just a list of Tuples, e.g., [(-3, 3), (-4, 2), ...(-7.5, 1.2)].
So one can either input this into the function or just override the output. There is no magic happening with respect to the integral bounds inside the test functions, i.e., the integrals bounds the user puts in would be the integral bounds which are returned by test function. What is the better option?

**Comment by apaleyes** [comment]
@mmahsereci how an "override output" code would look like?

**Comment by mmahsereci** [comment]
@apaleyes , I added the docstrings as discussed.

File: emukit/test_functions/quadrature/baselines.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+
+from typing import Tuple, List
+import numpy as np
+from scipy.integrate import quad, dblquad
+
+
+def univariate_approximate_ground_truth_integral(func, integral_bounds: Tuple[float, float]):
+    """
+    Uses scipy.integrate.quad to estimate the ground truth integral
```
Comment (apaleyes) [inline emukit/test_functions/quadrature/baselines.py]: Minor: this gives out implementation detail. Better phrasing would be "Estimate the ground truth integral." But for test functions this is very minor concern

File: emukit/test_functions/quadrature/baselines.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    :returns: integral estimate, output of scipy.integrate.quad
+    """
+    lower_bound = integral_bounds[0]
+    upper_bound = integral_bounds[1]
+    return quad(func, lower_bound, upper_bound)
+
+
+def bivariate_approximate_ground_truth_integral(func, integral_bounds: List[Tuple[float, float]]):
+    """
+    Uses scipy.integrate.dblquad to estimate the ground truth integral
```
Comment (apaleyes) [inline emukit/test_functions/quadrature/baselines.py]: ditto

File: emukit/test_functions/quadrature/baselines.py (hunk 1 above)
Comment (mmahsereci) [reply emukit/test_functions/quadrature/baselines.py]: sure, I'll rephrase.

**Pull request merged**